manage.py

```diff
@@ -58,6 +59,7 @@ def deploy():
     analyze_all()
 
 
+@boto3_mocks.restore_http_adapter
```

Comment:
Yep, I tried decorating individual tests, class-level, and explicit mock/unmock :(